public/login/login.html

```diff
@@ -18,7 +18,7 @@
             <li id="notification"><a href="#"></a></li>
         </ul>
     </header>
-    <form class="login_form" id="loginForm" method="GET" action="/">
+    <form class="login_form" id="loginForm" method="POST" action="/">
```

Comment:
Don't use form's functionality. Do it the same way you did the textsnippet.
Find all the buttons and fields and fetch (remember to preventDefault)